packages/aws-cdk-lib/aws-ec2/lib/vpc.ts

```diff
@@ -1508,6 +1633,11 @@ export class Vpc extends VpcBase {
 
       (this.publicSubnets as PublicSubnet[]).forEach(publicSubnet => {
         publicSubnet.addDefaultInternetRoute(igw.ref, att);
+
+        // configure IPv6 route if VPC is dual stack
+        if (this.useIpv6) {
+          publicSubnet.addIpv6DefaultInternetRoute(igw.ref);
```

Comment:
Suggest also adding a NAT64 route, to the NAT gateway (used in IPv4). This will allow IPv6 only machines to use DNS64 (e.g. Google) and be able to access IPv4-only resources. Just adding the route has no drawback.